models/user.go

```diff
@@ -86,6 +85,7 @@ func NewUser(instanceID uuid.UUID, email, password, aud string, userData map[str
 		ID:                id,
 		Aud:               aud,
 		Email:             storage.NullString(strings.ToLower(email)),
+		Phone:             storage.NullString(strings.ToLower(phone)),
```

Comment:
i don't think we'll need to convert the phone number to lowercase since they'll all be digits

Reply:
Ah yeah, great catch -- thanks!